frontend/src/pages/NewRunV2.tsx

```diff
@@ -78,6 +78,8 @@ interface RunV2Props {
   namespace?: string;
   existingRunId: string | null;
   apiRun: ApiRunDetail | undefined;
+  originalRecurringRunId: string | null;
+  apiRecurringRun: ApiJob | undefined;
```

Comment:
I think `apiRecurringRun` and `apiRun` should be exclusive to each other, that means if one is not empty, another one should be empty. Otherwise, a lot of inner logics of this component will yield error. For example `hasVersionId()`

Reply:
Yes, they should be exclusive to each other.